aquadoggo/src/db/stores/task.rs

```diff
@@ -62,8 +63,8 @@ impl SqlStorage {
             WHERE
                 name = $1
                 -- Use `IS` because these columns can contain `null` values.
-                AND document_id IS $2
-                AND document_view_id IS $3
+                AND COALESCE(document_id, '') = COALESCE($2, '')
```

Comment:
`IS` doesn't seem to be supported in postgres so this is a workaround to use `=`

Reply:
For some reason it did not allow me to use an empty string?